magenta-lib/src/main/scala/magenta/tasks/FastlyTasks.scala

```diff
@@ -30,43 +31,73 @@ case class UpdateFastlyConfig(s3Package: S3Path)(implicit
   implicit val ec: ExecutionContextExecutorService =
     ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(10))
 
-  override def execute(
+  private def _execute(
```

Comment:
Is there a more meaningful name for this method? Perhaps `executeWithCredentials`?